### PR TITLE
Added a bash script to test program-list.json order (and some fixes)

### PR DESCRIPTION
**Pull request by adiffpirate**
I added "Synology" program to the list below "Synack" to respect the alphabetical order, hope it makes your life easier @codingo

### CODE REVIEW

**Comment by adiffpirate** [comment]
Added a tool to adress issue #113. This is my first code contribuition to a project, it was pretty fun, so thank you for the encouragement @codingo. I gonna upgrade this tool later... Thinking in add a option to receive a json file via link, so that way you don't have to download every "program-list.json" file for every pull request. What do you think? Any other ideas?

**Comment by adiffpirate** [comment]
Also, I used the python script from #145 (with some little changes) to order the json file, but could'nt find a good way to compare two json's with in python, hences the bash script. Gonna give a look on that later too, and try to make it all in python :)

File: tools/Test JSON order/aux_test_order.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# Ensure proper arguments are given, and warn for missing ones.
+args_exit = False
+if args.jsonfile is None:
+    print("Error: input json file (-j/--jsonfile) is required")
+    args_exit = True
+if args.outfile_json is None:
+    print("Error: Output JSON filename (-oj/--outfile-json) is required")
+    args_exit = True
+if args_exit:
+    sys.exit(0)
```
Comment (mendesfabio) [inline tools/Test JSON order/aux_test_order.py:28]:
```suggestion
 if None in (args.jsonfile, args.outfile_csv, args.outfile_json):
    if args.jsonfile is None: 
        print("Error: input json file (-j/--jsonfile) is required")
    if args.outfile_csv is None:
        print("Error: Output CSV filename (-oc/--outfile-csv) is required")
    if args.outfile_json is None:
        print("Error: Output JSON filename (-oj/--outfile-json) is required")
    sys.exit(0)
```

Comment (mendesfabio) [inline tools/Test JSON order/aux_test_order.py:28]:
```suggestion
if None in (args.jsonfile, args.outfile_json):
    if args.jsonfile is None: 
        print("Error: input json file (-j/--jsonfile) is required")
    if args.outfile_json is None:
        print("Error: Output JSON filename (-oj/--outfile-json) is required")
    sys.exit(0)
```

Comment (mendesfabio) [reply tools/Test JSON order/aux_test_order.py:28]: Consider this suggestion! I did it on #148.

**Comment by adiffpirate** [comment]
Oh, nice one @mendesfabio, ty for that

**Comment by adiffpirate** [comment]
Removed the Synology program I added earlier, because it was added on #150

**Comment by codingo** [comment]
@adiffpirate could you please resolve the conflicts here and then I'll revisit?

**Comment by adiffpirate** [comment]
I think it's all resolved @codingo

**Comment by adiffpirate** [comment]
Sorry for bother you @codingo, but I just remembered about your tweet last week and wondered if that offer applies in my case 😅 Again, sorry if I'm being indiscreet or something like that

**Comment by codingo** [comment]
I was confident I'd already sent you one in another pull request but I can't find a reference. Could you let me know your twitter account and I'll send you a voucher?

**Comment by adiffpirate** [comment]
[Here](https://twitter.com/adiffpirate) sir, thank you so much ❤️

**Comment by codingo** [comment]
Thank-you, sent to you now!

**Pull request merged**